packages/main/src/RPA/Desktop/__init__.py

```diff
@@ -44,28 +46,45 @@ class Desktop(DynamicCore):
 
     The currently supported locator types are:
 
-    =========== =================== ===========
-    Name        Arguments           Description
-    =========== =================== ===========
-    alias       name (str)          A named locator, the default.
-    image       path (str)          Image of an element that is matched to current screen content
-    coordinates x (int), y (int)    Pixel coordinates as absolute position
-    offset      x (int), y (int)    Pixel coordinates relative to current mouse position
-    =========== =================== ===========
+    =========== ================================================ ===========
+    Name        Arguments                                        Description
+    =========== ================================================ ===========
+    alias       name (str)                                       A custom named locator from the locator database, the default.
+    image       path (str)                                       Image of an element that is matched to current screen content.
+    point       x (int), y (int)                                 Pixel coordinates as absolute position.
+    offset      x (int), y (int)                                 Pixel coordinates relative to current mouse position.
+    region      left (int), top (int), right (int), bottom (int) Bounding coordinates for a rectangular region.
+    ocr         text (str), confidence (float, optional)         Text to find from the current screen.
+    =========== ================================================ ===========
 
     A locator is defined by its type and arguments, divided by a colon.
-    Some example usages are shown below. Note that the prefix for alias can
+    Some example usages are shown below. Note that the prefix for ``alias`` can
     be omitted as its the default type.
 
     .. code-block:: robotframework
 
-        Click    alias:SpareBin.Login
-        Click    SpareBin.Login
+        Click       alias:SpareBin.Login
+        Click       SpareBin.Login
 
-        Move mouse    image:%{ROBOT_ROOT}/logo.png
-        Move mouse    offset:200,0
+        Move mouse  image:%{ROBOT_ROOT}/logo.png
+        Move mouse  offset:200,0
         Click
 
+        Click       point:50,100
+
+        Click       region:20,20,100,30
+
+        Click       ocr:Create New Account
```

Comment:
broken with current master. when #110 is merged, this would work as `ocr:"Create New Account"`